### DISCUSSION TITLE
[4.x] - Filters should be savable, reordable, duplicatable etc.

### DISCUSSION THREAD

**Discussion by outline4** [craftcms/cms#11215] [Ideas] [answered]
Something like the good old Zenbu from Expressionengine.  
They did a really great job at filtering.  
I miss that since the beginning of Craft.  
There was a version once, but it was badly implemented and way overpriced. 

Go Brandon and team!

**Comment by brandonkelly** [comment, marked as answer]
We do plan on making filters savable/sharable. Doubt we’ll make it possible to duplicate though.

**Reply by fernakr** [reply]
Just chiming in to say that a client just requested this so looking forward to seeing this!

**Comment by brandonkelly** [comment]
Craft 4.5.0 is out with an improvement related to this: filters are now serialized and added to the query string, so they will persist when you navigate to an entry and then press the Back button, or if you share the index page URL with a colleague.

**Reply by outline4** [reply]
Imagine an online shop, you carefully choose your filter (this might involve several steps), you klick on one of the products. In the detail product page you klick on the back button (not the browser back button) and you land on the landing page with unfiltered results. That's kind of frustrating, isn't it. 

Same feeling here.

**Reply by sabrallte** [reply]
Yep, same here. Hope there will be savable filters soon. But anyway, thank you for Craft CMS!
